src/pool/srv_iv.c

```diff
@@ -1066,6 +1078,7 @@ pool_iv_update(void *ns, int class_id, uuid_t key_uuid,
 	key.class_id = class_id;
 	pool_key = (struct pool_iv_key *)key.key_buf;
 	pool_key->pik_entry_size = pool_iv_len;
+	pool_key->pik_eph = crt_hlc_get();
```

Comment:
Talked with Di offline. Seems this pik_eph is specific to pool prop IV, and there is no pool prop IV invalidation, so pik_eph isn't set on invalidate path.

I'm ok with landing it as-is if it can fix an urgent bug, but I think the code needs be reorganized a bit in the future.